packages/web-app/src/views/appeal-details/application-decision-date.njk

```diff
@@ -0,0 +1,54 @@
+{% extends "layouts/two-thirds-page.njk" %}
```

Comment:
This template should extend the form layout as this will reduce the code required in this template